src/containers/ViewPage/ViewPage.js

```diff
@@ -86,33 +72,28 @@ const ViewPage = ({ games, platforms, fetchGames, fetchPlatforms, changeTitleAct
   );
 };
 
-const mapStateToProps = (store) => {
-  return {
-    games: store.games.games,
-    platforms: store.platforms.platforms
-  }
-};
+const mapStateToProps = (store) => ({
+  categoryData: store.category.categoryData
```

Comment:
if you have category, inside data should be simply called data, but not categoryData